pkg/kv/kvserver/store_create_replica.go

```diff
@@ -93,6 +93,67 @@ func (s *Store) getOrCreateReplica(
 	}
 }
 
+func (s *Store) tryGetReplica(
```

Comment:
Could you give this method a comment still? It's unusual to return `nil, nil` so we should call that out.

Reply:
I'm a little sad that now we have three methods in this dance. That a method called `tryGetReplica` can replicaGC a replica is also a little odd. (It was odd before in `tryGetOrCreateReplica` but at least that method seemed like it could have side effects).

Reply:
I think it would make sense to inline this if we can. Having lots of random methods on `Store` that only work in one very specific situation isn't helpful. I don't love inlining it either though. If you can't see any better option I'm fine leaving as is.

We could move these methods off the `*Store` receiver (taking the store as a parameter instead), maybe that makes the special-purposeness of this a bit clearer.